schemas/answers/text_field.json

```diff
@@ -19,8 +19,21 @@
       "description": {
         "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/string_with_placeholders"
       },
-      "suggestions_url": {
-        "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+      "suggestions": {
+        "type": "object",
+        "properties": {
+          "url": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
+          },
+          "allow_multiple": {
+            "type": "boolean"
+          }
+        },
+        "dependencies": {
+          "suggestions": {
```

Comment:
I don't think dependencies is needed as `url` is nested inside `suggestions`. Just marking `url` as required should only make it mandatory if the optional `suggestions` property exists.

Reply:
I marked `url` as required now, dependency removed. Thanks!